next.config.js

```diff
@@ -29,6 +27,11 @@ module.exports = withPlugins([
       use: {loader: '@svgr/webpack'}
     })
 
+    config.module.rules.push({
```

Comment:
Does this mean that everyone consuming Primer Components will need to add a loader for CSS files?

Reply:
No! I specifically built it so that the CSS is resolved at build-time by rollup and insertred as a string into the final bundle.

That string is then injected via styled-components `createGlobalStyle`, which means users do not need to configure anything :+1: